tests/test_sysroot_creator.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
```

Comment:
Unit tests should generally not touch the file system. Is this the case here?
If yes, consider using pyfakefs (https://github.com/jmcgeheeiv/pyfakefs).